packages/labs/ssr-client/src/lit-element-hydrate-support.ts

```diff
@@ -0,0 +1,115 @@
+/**
+ * @license
+ * Copyright 2017 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+/**
+ * LitElement support for hydration of content rendered using lit-ssr.
+ *
+ * @packageDocumentation
+ */
+
+import type {PropertyValues} from '@lit/reactive-element';
+import {render, RenderOptions} from 'lit-html';
+import {hydrate} from 'lit-html/experimental-hydrate.js';
+import {HYDRATE_INTERNALS_ATTR_PREFIX} from '@lit-labs/ssr-dom-shim';
```

Comment:
I wish I had caught this in the SSR-internals code review. I don't think the client packages should be depending on the shim package like this. I think we can define this attribute a bit like a protocol and just define it in multiple places so we don't need this dependency. At least I'd want to make a separate module for this in ssr-dom-shim.

Reply:
Makse sense. Filed https://github.com/lit/lit/issues/3769

Reply:
Can you just copy the value into here now so that this package doesn't need to depend on the ssr-dom-shim?